site/app/templates/grading/electronic/PDFAnnotationEmbedded.twig

```diff
@@ -67,12 +67,17 @@
     same could not be achived through CSS as it cause the tools to hide when moved right in panel.
     This has been done taking care of resizing of panel ,platform(device),overlapping and scrolling of panels */
 
-    const offSetter = new ResizeObserver(entries => {
-        entries.forEach(entry => {
-            let height = $(".sticky-file-info").outerHeight();
-            $("#viewer").css("paddingTop", height);
+    try {
+        const offSetter = new ResizeObserver(entries => {
+            entries.forEach(entry => {
+                let height = $(".sticky-file-info").outerHeight();
+                $("#viewer").css("paddingTop", height);
```

Comment:
Is this line doing anything productive?  I believe that the correct way to set the padding on the top would be 
`$("#viewer").css("padding-top", height);`.